src/ui/components/Icon/plus.svg

```diff
@@ -0,0 +1,27 @@
+<svg width="48px" height="48px" viewBox="12 6 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 47.1 (45422) - http://www.bohemiancoding.com/sketch -->
+    <desc>Created with Sketch.</desc>
+    <defs>
+        <filter x="-11.9%" y="-36.7%" width="123.7%" height="186.7%" filterUnits="objectBoundingBox" id="filter-1">
```

Comment:
This looks very complicated for a plus sign 😛

Reply:
Yeah, it's copied from Sketch 🤷🏻‍♂️